src/flags/Flags.ts

```diff
@@ -62,6 +62,15 @@ export const gitRemoteFlags = {
   }),
 }
 
+export const gitCommitMessageFlag = {
+  commitMessage: flags.string({
```

Comment:
I think that a better name would be `buildVersionsMessage` because there is still a chance that Build Versions could be extended to support other backends, for example KV

I also think this should be a nullable field, and the default, should be set in the Build Versions Provider implementation.  This will allow us to discern from an actual null flag, and leaves room for the Build Versions Provider to auto-generate some useful text.